scripts/update-1.6.1-next_version.sql

```diff
@@ -0,0 +1,18 @@
+-- This script contains multiple comment blocks that can update
+-- APEL version 1.6.1 databases of the following types to the next version:
+--  - Cloud Accounting Database
+
+-- UPDATE SCRIPT FOR CLOUD SCHEMA
+
+-- If you have a Cloud Accounting Database and wish to
+-- upgrade to APEL Version next, remove the block comment
+-- symbols around this section and run this script
+
+-- This section will:
+-- - Change records with a NULL CpuCount so that they have a CpuCount of 0,
+--   to prevent summariser time problems.
```

Comment:
Little ambiguous. You mean problems at summarising time, right?

Reply:
Yes, change made